cmd/manager-hub/main.go

```diff
@@ -139,6 +139,10 @@ func main() {
 		cmd.FatalError(ctrlLogger, err, "unable to create controller")
 	}
 
+	if err = (&v1beta1.ManagedClusterModule{}).SetupWebhookWithManager(mgr); err != nil {
```

Comment:
WDYT about adding a comment regarding the validation path explanation ( what we found out today during debug) to some place in a code

Reply:
Added the comment inside `SetupWebhookWithManager` for both `Module` and `ManagedClusterModule` webhook files.